compass/landice/tests/ismip6_run/ismip6_ais_proj2300/ismip6_ais_proj2300.cfg

```
@@ -44,3 +44,19 @@ von_mises_parameter_path = UNKNOWN
 
 # Whether facemelting should be included in the runs
 use_face_melting = False
+
+# True if running coupled MALI-sea level model simulation
+sea_level_model = True
```

Let's leave the default as False for now to keep the behavior from the submitted ISMIP6 experiments.
```suggestion
sea_level_model = False
```